src/Infrastructure/PiwikCustomTracking.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * Simple wrapper for tracking methods in Piwik's API
+ *
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class PiwikCustomTracking {
+
+	const SCOPE_VISIT = 'visit';
+	const SCOPE_PAGE = 'page';
```

Comment:
Did you intend to use these in the route handlers? As far as I can tell they are unused now.